cmd/ocm-backplane/cloud/console.go

```diff
@@ -132,13 +134,39 @@ func runConsole(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	// ======== Get cloud console from backplane API ============
-	response, err := getCloudConsole(bpURL, clusterID)
+	stsSupportJumpRole, err := utils.DefaultOCMInterface.GetStsSupportJumpRole(clusterID)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get sts support jump role: %w", err)
+	}
+	supportRoleArn, err := arn.Parse(stsSupportJumpRole)
+	if err != nil {
+		return fmt.Errorf("failed to parse ARN for jump role %v: %w", stsSupportJumpRole, err)
+	}
+
+	var consoleResponse *ConsoleResponse
+	if supportRoleArn.Resource == OldFlowSupportRole { // cluster is using old backplane flow
+		consoleResponse, err = getCloudConsole(bpURL, clusterID)
+		if err != nil {
+			return err
+		}
+	} else { // cluster is using new backplane flow
+		targetCredentials, err := getNewFlowCredential(clusterID)
```

Comment:
Minor Nitpick: Could we rename the function to just `getCloudCredentials` or something? I feel like the name `NewFlow` will get *old* pretty quickly :)

Reply:
I agree with this, and I struggled to find a good name because `getCloudCredential` already exists in this package in `credentials.go`. Any other ideas on a name?

Reply:
Thought of a better name - coming in a commit.